gossip/src/crds_value.rs

```diff
@@ -204,6 +206,39 @@ impl SnapshotHash {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, AbiExample)]
+pub struct IncrementalSnapshotHash {
+    pub base_slot: Slot,
+    pub snapshot_hash: SnapshotHash,
```

Comment:
nit: I'd rather we inline `SnapshotHash` here for clarity.

Reply:
Sure thing, will do!

Reply:
Done!